app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -75,6 +80,19 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
         initSession()
     }
 
+    @SuppressLint("RestrictedApi")
+    override fun onResume() {
+        super.onResume()
+        if (session.isYoutubeTV) {
+            // Send key events on the UI thread to clear webview grey screen, see #1865
+            GlobalScope.launch(Dispatchers.Main) {
+                delay(50)
+                activity?.dispatchKeyEvent(KeyEvent(KeyEvent.ACTION_DOWN, KeyEvent.KEYCODE_DPAD_RIGHT))
+                activity?.dispatchKeyEvent(KeyEvent(KeyEvent.ACTION_DOWN, KeyEvent.KEYCODE_DPAD_LEFT))
```

Comment:
Nit: I think we could more clearly communicate what this is doing by breaking it into a new class (something like `YouTubeGreyScreenWorkaround#invoke`).  The code is pretty trivial, but it would 1) let us add a kdoc with some relevant context, and 2) clearly mark it is a workaround.